examples/image-component/package.json

```
@@ -9,5 +9,10 @@
     "next": "latest",
     "react": "^17.0.2",
     "react-dom": "^17.0.2"
```

Could we bump these?
```suggestion
    "react": "^18.2.0",
    "react-dom": "^18.2.0"
```